lib/modules/dosomething/dosomething_global/dosomething_global.module

```diff
@@ -220,7 +220,7 @@ function dosomething_global_convert_country_to_language($country) {
       return $lang_key;
     }
   }
-  return 'en';
+  return NULL;
```

Comment:
yep- just to add it here first, 

this function shouldn't be deciding what the correct default value should be, as the name implies its only purpose is to find the matching language for that country. In some cases, that country doesn't have a matching language and it's useful to have it return a default lang, in others, it might be more preferable to have it return NULL.